### PR TITLE
feat: modify inputs for group functionality (#375)

### PR DESCRIPTION
**Pull request by johglove**

### CODE REVIEW

**Review by burnumd — CHANGES_REQUESTED** [review]
As a reminder, commits must follow the [Angular Convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for the release process to work

**Comment by johglove** [comment]
What are the conventions not being followed?

**Comment by burnumd** [comment]
> What are the conventions not being followed?

This PR, as an example, probably needs to be a `feat: modify inputs for group functionality (#375)`.
If it doesn't have a commit type, it will not trigger a release and the commit will not be included in the release notes.

File: src/components/Input/common.jsx [hunk 1]
```diff
@@ -46,22 +54,29 @@ export const renderInput = (
...
     ...attrs
   }
 ) => {
   const noteId = `${id}_note`;
+  const groupped = appendment ||  prependment;
+
   const inputProps = {
     id,
-    className: inputClassName(elementName, variant),
+    className: inputClassName(elementName, variant, groupped),
```
Comment (burnumd) [inline src/components/Input/common.jsx:72]:
```suggestion
    className: inputClassName(elementName, variant, grouped),
```

File: src/components/Input/common.jsx [hunk 2]
```diff
@@ -73,7 +88,15 @@ export const renderInput = (
       >
         {label}
       </label>
-      {React.createElement(elementName, inputProps)}
+      {!groupped && inputElement}
```
Comment (burnumd) [inline src/components/Input/common.jsx:91]:
```suggestion
      {!grouped && inputElement}
```

**Review by burnumd — CHANGES_REQUESTED** [review]

**Review by burnumd — APPROVED** [review]

**Pull request merged**